.circleci/config.yml

```diff
@@ -718,28 +688,48 @@ jobs:
 
 
   perf_test:
-    executor: linux-amd64
+    docker:
+      - image: cimg/base:2021.04
+    resource_class: small
     parameters:
-      format:
-        type: string
-      record_ingest_results:
+      record_results:
```

Comment:
Is `record_results` used for anything other than CI update dry runs?

Reply:
nope